app/models/form.rb

```diff
@@ -3,18 +3,18 @@ class Form
               :start_date, :end_date, :type, :name, :setup_definition,
               :setup_sections, :form_sections
 
-  def initialize(form_path, name, sections_in_form = [], type = "lettings")
+  def initialize(form_path, start_year = "", sections_in_form = [], type = "lettings")
```

Comment:
```suggestion
  def initialize(form_path, start_year, sections_in_form = [], type = "lettings")
```

Is `start_year` optional?

Reply:
I think so. We get the start year for lettings form from the JSON file at the moment, so we wouldn't have it before we create the lettings form (line 35, same file)

Reply:
Ah, that makes sense, because we need to support a manual initialization _and_ taking the year from the form. Thanks!